src/main/resources/db/migration/V1__create_tables.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TYPE PHONE_TYPE as ENUM ('home', 'work', 'mobile');
+
+CREATE TYPE ROLE as ENUM ('associate', 'partner', 'senior');
+
+CREATE TYPE STATUS as ENUM ('new', 'assigned', 'on_hold',
+    'approved', 'canceled', 'declined');
+
+CREATE TABLE users
+(
+    id               bigserial
+        constraint PK_users PRIMARY KEY,
+    email            text not null
+        constraint UQ_users_email UNIQUE ,
+    cognito_username text not null
+        constraint UQ_users_cognito_username UNIQUE
+);
+
+CREATE TABLE advisors
+(
+    id   bigserial
+        constraint PK_advisors_id PRIMARY KEY,
+    role ROLE not null,
+    constraint advisor_users_fk FOREIGN KEY (id)
+        references users (id)
+);
+
+CREATE TABLE applicants
+(
+    id         bigserial
+        constraint PK_applicants PRIMARY KEY,
+    first_name text not null,
+    last_name  text not null,
+    ssn        int  not null
+        constraint UQ_applicants_ssn UNIQUE,
+    constraint applicants_users_fk FOREIGN KEY (id)
+        references users (id)
+);
+
+CREATE TABLE numbers
+(
+    id           bigserial
+        constraint PK_numbers PRIMARY KEY,
+    number       text       not null,
```

Comment:
number is indeed a reserved keyword. However, when it comes to creating an enumeration type (enum) named "number", you can still do so, but you need to handle it with quotes ""